no-computer.md

```diff
@@ -6,39 +6,75 @@ This section describes who the project would serve and why a language might be a
+the movements. This is more aimed at beginner level dancers than advanced
+dancers who have master most of the movement. By having this broken down as a
+language, it can help beginning dancers organize the different movements they
+have available to them and when it is appropriate to use them.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate because of the strict nature of West Coast Swing technique.
+Unlike other social dances, the vocabulary of movement has more definition
+and expectations as to how it's executed, especially at a competition level. 
+Understanding and remembering all of the different pieces (and what is acceptable
+when) can be a lot to manage at the beginner level. By understanding this information
+as a language, it can be easier understood and manipulated.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I've been dancing west coast swing for a while, and I still feel a bit of thrill
+when dancing it or watching it. I came up with this by trying to think of everday
+activities that have specific rules to them, and this came to mind.
 
 ### Domain
 _Describe the project's domain in five words._
 
+West Coast Swing, partner dancing.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user interacts with this language through dancing, either by practicing
+on their own or by dancing with a partner. Programming looks like the set of
+moves they plan to execute or a choreographed routine. It can also look like
+a combination they are practicing. This is a good way to start picking up the
+basics of a dance--by learning premade sets--so this would be a useful way to 
+begin learning west coast swing.
+
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+When a program runs, a dance happens. The program interacts with the user in 
+that the user has to physically carry it out. Errors include not remembering
+the steps, not following the basic rythym, not being on time, stepping on 
+your partner's feet, etc. Usually, an outside observer can make note of these.
+Users can also videotape themselves dancing to see the mistakes that they make.
+
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+Executing a sequence of west coast moves should be possible in this language.
+Given the direction of momentum required to execute some of these movements
+must be provided by the previous movement, some movements may not flow very
+naturally together. Dancing tango should be impossible.
 
```

Comment:
It seems like you would need to severely limit the vocabulary so that the language could not be misused for other dances.